Library/Homebrew/utils/github.rb

```diff
@@ -666,4 +666,37 @@ def pull_request_labels(user, repo, pr)
     pr_data = API.open_rest(url_to("repos", user, repo, "pulls", pr))
     pr_data["labels"].map { |label| label["name"] }
   end
+
+  def last_commit(user, repo, ref)
+    return if Homebrew::EnvConfig.no_github_api?
+
+    output, _, status = curl_output(
+      "--silent", "--head", "--location",
+      "--header", "Accept: application/vnd.github.sha",
+      url_to("repos", user, repo, "commits", ref).to_s
+    )
+
+    return unless status.success?
+
+    commit = output[/^ETag: "(\h+)"/, 1]
+    return if commit.blank?
+
+    version.update_commit(commit)
+    commit
+  end
+
+  def multiple_short_commits_exist?(user, repo, commit)
+    return if Homebrew::EnvConfig.no_github_api?
+
+    output, _, status = curl_output(
+      "--silent", "--head", "--location",
+      "--header", "Accept: application/vnd.github.sha",
+      url_to("repos", user, repo, "commits", commit).to_s
+    )
+
+    return true if status.success?
+    return true unless output
```

Comment:
```suggestion
    return true unless status.success?
    return true if output.blank?
```

I'm pretty sure?

Reply:
It's right, the original code `!(status.success? && output && output[/^Status: (200)/, 1] == "200")` is equal to `!status.success? || !output || output[/^Status: (200)/, 1] != "200"`, therefore `return true unless status.success?` is right.